src/pages/roster/index.tsx

```diff
@@ -0,0 +1,29 @@
+import useStore from '@hooks/useStore';
+import { observer } from 'mobx-react-lite';
+import { useEffect } from 'react';
+import AddCharacterForm from '@components/AddCharacterForm';
+
+const Roster = () => {
+  const { roster } = useStore();
+
+  useEffect(() => {
+    roster.getRoster();
+  }, []);
+
+  const characters = roster.characters.map((character) => {
+    return (
+      <div className="character" key={`${character.name}_${character.realm}`}>
+        {character.name} - {character.realm}
+      </div>
+    );
+  });
```

Comment:
I'd suggest to create another component for this map so instead of calling this function we can do

```javascript

const characters = roster.characters.map(character => <Character name={name} realm={realm} />
```